packages/breadcrumb/src/react/index.tsx

```diff
@@ -0,0 +1,58 @@
+import Button from '@pluralsight/ps-design-system-button'
+import { CaretLeftIcon } from '@pluralsight/ps-design-system-icon'
+import { RefForwardingComponent } from '@pluralsight/ps-design-system-util'
+import { css } from 'glamor'
+import React from 'react'
+
+import stylesheet from '../css'
+
+const styles = {
+  breadcrumb: () => css(stylesheet['.psds-breadcrumb'])
+}
+
+interface BreadcrumbProps
+  extends Omit<
+    React.HTMLAttributes<HTMLDivElement>,
+    'disabled' | 'href' | 'onClick'
```

Comment:
Also if we dropped the wrapping div we wouldn't have to omit these

Reply:
Hey, I think the point on the surrounding div is a fair point.  I felt that myself as I was realizing what we had done there.  But for now, I'm just wanting to type what is there with minimal collateral damage.